examples/cloudtube.config.js

```diff
@@ -1,13 +1,16 @@
 module.exports = {
-	/*
-	  Copy this file to `config.js`, and add options here.
-	  They'll override the options from `utils/constants.js`.
-	  For example, the next block changes the default instance.
-	*/
+        /*
+          Copy this file to `config.js`, and add options here.
+          They'll override the options from `utils/constants.js`.
+          For example, the next block changes the default instance.
+        */
 
-	user_settings: {
-		instance: {
-			default: "http://newleaf:3000"
-		}
-	}
-}
\ No newline at end of file
+        user_settings: {
+                instance: {
+                        default: "http://newleaf:3000"
+                }
+        },
+        server_setup: {
+                local_instance_origin: "http://newleaf:3000"
```

Comment:
Sorry to bother you, can you add a short comment here saying why this variable is set? Looks good to be merged otherwise.